CMakeLists.txt

```diff
@@ -4,7 +4,18 @@ cmake_minimum_required(VERSION 3.4)
 
 include(ExternalProject)
 
-project( CCTag )
+project(CCTag VERSION 1.0.0 LANGUAGES C CXX)
```

Comment:
> So your IDE tools will make an error 
Nitpicking, but true only for the IDEs that do not support cmake, the file is generated during the cmake configuration and it is enough to include it among the include directories of the targets with e.g. `$<BUILD_INTERFACE:<the place where generated files are>>` (so that it only works for in-tree build), at that point the IDE has no problem at retrieving it.